sktime/datasets/_single_problem_loaders.py

```diff
@@ -863,3 +868,40 @@ def load_macroeconomic():
     y = y.drop(columns=["year", "quarter", "time"])
     y.name = "US Macroeconomic Data"
     return y
+
+
+def load_unit_test_tsf():
+    """
+    Load tsf UnitTest dataset.
+
+    Returns
+    -------
+    loaded_data: pd.DataFrame
+        The converted dataframe containing the time series.
+    frequency: str
+        The frequency of the dataset.
+    forecast_horizon: int
+        The expected forecast horizon of the dataset.
+    contain_missing_values: bool
+        Whether the dataset contains missing values or not.
+    contain_equal_length: bool
+        Whether the series have equal lengths or not.
+    """
+    name = "UnitTest"
```

Comment:
it seems that the local variables are not rally necessary here

Reply:
Removed the extra local variables.